src/Migration/Migration1694519020TransportFailed.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Frosh\MailArchive\Migration;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+class Migration1694519020TransportFailed extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1694519020;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeStatement('ALTER TABLE `frosh_mail_archive` ADD `transport_failed` TINYINT(1) NOT NULL DEFAULT 0 AFTER `customerId`;');
```

Comment:
can you remove the `after` in your query? After making that, the whole table needs to be copied.

Reply:
Sure thing. Here you go.